public/bower_components/angular-bootstrap/.bower.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
@tremainebuchanan exclude the `bower_components` directory from git using the `.gitignore` file.